src/sentry/migrations/0170_cool_users.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 1.11.29 on 2021-03-03 06:02
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = False
+
+    # This flag is used to decide whether to run this migration in a transaction or not.
+    # By default we prefer to run in a transaction, but for migrations where you want
+    # to `CREATE INDEX CONCURRENTLY` this needs to be set to False. Typically you'll
+    # want to create an index concurrently when adding one to an existing table.
+    # You'll also usually want to set this to `False` if you're writing a data
+    # migration, since we don't want the entire migration to run in one long-running
+    # transaction.
+    atomic = True
+
+    dependencies = [
+        ("sentry", "0169_delete_organization_integration_from_projectcodeowners"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="user",
+            name="last_active",
+        ),
+        migrations.AddField(
+            model_name="user",
+            name="is_cool",
+            field=models.BooleanField(default=True),
```

Comment:
Backfilling `True` as part of the migration may be too slow and leave the table locked for too long. If so, consider initializing the new column to null, while having the model default to `True` for new rows. Then, we can backfill in a later migration (non-locking? I'm not clear on the details) and make the column non-nullable when it's safe.

In order to follow the above suggestion, from the time of the migration until the backfill is complete, any new code that reads the `is_cool` value would have to check for both `None` and `True` and treat them as equivalent. Defining `None` to be semantically truthy (where Python treats it as falsy unless explicitly looked for with `is None`) is ugly and invites bugs. Consider inverting the new field's semantics to `is_uncool` and defaulting to false/none.